prboom2/doc/mbf21.md

```diff
@@ -309,6 +340,22 @@ MBF21 defaults:
     - This function uses the same approximate distance check as other functions in the game (P_AproxDistance).
     - The jump will only occur if distance is BELOW the given value -- e.g. `A_JumpIfTargetCloser(420, 69)` will jump to state 420 if distance is 68 or lower.
 
+- **A_JumpIfTracerInSight(state)**
+  - Jumps to a state if caller's tracer (seek target) is in line-of-sight.
+  - Args:
+    - `state (uint)`: State to jump to
+  - Notes:
+    - This function only considers line-of-sight, i.e. independent of calling actor's facing direction.
```

Comment:
Should we add an fov arg here as well for completeness?

Reply:
May as well, now that we have the code for that written somewhere :P -- I'd like to do that in a separate PR though, if that's OK, since A_JumpIfTargetInSight needs it too (and I gotta fix a crash bug with that one anyway, lol).

Reply:
Alrighty